Vagrantfile

```diff
@@ -0,0 +1,34 @@
+# -*- mode: ruby -*-
```

Comment:
Is it possible to move this somewhere under hack/?

Where possible we've tried to avoid cluttering the repo root for better discovery on GitHub and easier delegation of OWNERS rights.

Reply:
moved